src/css/controls/imports/display.less

```diff
@@ -90,11 +90,15 @@ display icons
 }
 
 .jw-breakpoint-1 {
+    .jw-display-icon-container {
+        margin: 0 (@mobile-touch-target * 0.5);
+    }
+
     .jw-display .jw-icon,
     .jw-display .jw-svg-icon {
-        height: (@mobile-touch-target * 1.25);
-        line-height: (@mobile-touch-target * 1.25);
-        width: (@mobile-touch-target * 1.25);
+        height: (@mobile-touch-target);
+        line-height: (@mobile-touch-target);
+        width: (@mobile-touch-target);
```

Comment:
`padding: (@mobile-touch-target * 0.25);`

Reply:
and please put width and height together. idiomatic CSS property ordering is better than alphabetical https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Reply:
Agreed 👍